pkgs/tools/security/sequoia-sqv/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv
+, fetchFromGitLab
+, lib
+, nettle
+, nix-update-script
+, rustPlatform
+, pkg-config
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sequoia-sqv";
+  version = "1.1.0";
+
+  src = fetchFromGitLab {
+    owner = "sequoia-pgp";
+    repo = "sequoia-sqv";
+    rev = "v${version}";
+    hash = "sha256-KoB9YnPNE2aB5MW5G9r6Bk+1QnANVSKA2dp3ufSJ44M=";
+  };
+
+  cargoHash = "sha256-uwOU/yyh3eoD10El7Oe9E97F3dvPuXMHQhpnWEJ1gnI=";
+
+  nativeBuildInputs = [
+    pkg-config
+    rustPlatform.bindgenHook
+  ];
+
+  buildInputs = [
+    nettle
+  ];
+
+  # Install shell completion files
+  postInstall = ''
+    make PREFIX=$out install
+  '';
```

Comment:
this causes the install phase to completely rebuild everything that was built in the build phase. `nix log /nix/store/his5anch2jgm1cvxa15p530n9bcys9q1-sequoia-sqv-1.1.0` if you still have the result cached.

i think _maybe_ this goes away if you invoke make with `CARGO_TARGET_DIR=<something>` so that the Makefile knows about the stuff that nix built in the buildPhase. probably `CARGO_FLAGS` has to match too if you do that. might be simpler to just use normal `stdenv.mkDerivation` and let the upstream Makefile handle all the phases.

Reply:
Thanks for the ideas. Indeed it's correct to do it this way. I ended up finding those shell files and installing them myself.